plugin/EstateList.php

```diff
@@ -99,6 +102,12 @@ class EstateList
 	/** @var EstateDetailUrl */
 	private $_pLanguageSwitcher;
 
+	/** @var Redirector */
+	private $_redirectIfOldUrl;
+
+	/** @var bool */
+	private $_isOverride_DataView = false;
```

Comment:
I was hoping this wouldn't be necessary, especially it only exists for the tests